frontend/static/frontend/img/evictionfree/forms.svg

```diff
@@ -0,0 +1,25 @@
+<svg width="444" height="528" viewBox="0 0 444 528" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Hmm, this appears to largely be a PNG embedded in a SVG using a base64 encoding, which is likely to be bigger than just having a PNG (it's 204K)... maybe we should replace it with a PNG?

Reply:
Oh good catch! will fix in my next PR